data/styles/clear/include/priorities_4_overlays.prio.txt

```diff
@@ -1095,6 +1099,7 @@ amenity-vending_machine-fuel                        # icon z18- (also has captio
 amenity-vending_machine-parking_tickets             # icon z17- (also has caption(optional) z17-)
 amenity-vending_machine-public_transport_tickets    # icon z17- (also has caption(optional) z17-)
 amenity-waste_transfer_station                      # icon z17- (also has caption(optional) z19-)
+entrance-main                                       # icon z18- (also has caption(optional) z18-)
```

Comment:
why did you raise entrances so much?

Reply:
This was done for this case: 

![Anchors_at_entrance,_National_Maritime_Museum,_Greenwich_-_geograph org uk_-_3663313](https://github.com/organicmaps/organicmaps/assets/26939824/07e23414-24cc-4667-b1cd-1e2bb06213c2)
(https://www.openstreetmap.org/node/2541359479)

The anchors had a higher priority than the entrance, but the entrance is more useful so should be higher.


Returned entrances to a lower priority now 👍